Developers/Writing_Bears.rst

```diff
@@ -150,6 +150,36 @@ This will show the user a bunch of information related to the bear like:
 - A description of what the bear does - The sections which uses it - The
 settings it uses (optional and required)
 
+.. note::
+
+    The bears are not yet installed. We still have to specify
+    the bear directory using ``-d`` or ``--bear-dirs`` flag.
+
+
+Install locally Written Bears
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Let's say NewBear.py contain our NewBear. To install our NewBear :
```

Comment:
I don't like how this sounds. Maybe `Let's say that we wrote a file NewBear.py that contain our NewBear and we want to run it locally. To install our NewBear :`